config/config-example.js

```diff
@@ -157,6 +157,7 @@ exports.loguserstats = 1000 * 60 * 10; // 10 minutes
 // amount of traffic (i.e. hundreds of concurrent battles).
 exports.validatorprocesses = 1;
 exports.simulatorprocesses = 1;
+exports.verifierprocesses = 1;
```

Comment:
I didn't mean hardcode 1, I meant default to one. You can still load it from Config and everything, just do the usual thing of ``Config.verifierprocesses || 1``